src/EditorFeatures/Core/Implementation/AutomaticCompletion/AbstractAutomaticLineEnderCommandHandler.cs

```diff
@@ -58,6 +63,22 @@ public VSCommanding.CommandState GetCommandState(AutomaticLineEnderCommandArgs a
 
         public void ExecuteCommand(AutomaticLineEnderCommandArgs args, Action nextHandler, CommandExecutionContext context)
         {
+            // Completion will only be active here if this command wasn't handled by the completion controller itself.
+            if (_asyncCompletionBroker.IsCompletionActive(args.TextView))
+            {
+                var session = _asyncCompletionBroker.GetSession(args.TextView);
+                var computedItems = session.GetComputedItems(context.OperationContext.UserCancellationToken);
+                var softSelection = computedItems.SuggestionItemSelected || computedItems.UsesSoftSelection;
+                var behavior = session.Commit('\n', context.OperationContext.UserCancellationToken);
+                session.Dismiss();
+
```

Comment:
This fix look weird. It seems that the old completion (Roslyn) supports IChainedCommandHandler<AutomaticLineEnderCommandArgs> but the new one (Editor) does not support it.

Maybe we should make a fi on the Editor side?

Reply:
the fix definitely belongs on the editor side

Reply:
This fix can be reverted if the editor side fixes the issue, but it will not break the functionality to use it as a temporary workaround.